src/spec/create_extension_spec.py

```diff
@@ -365,7 +339,39 @@ def main():
         ]
     )
 
-    # TODO: add all of your new data types to this list
+    fiberphotometryresponse_series = NWBGroupSpec(
+        neurodata_type_def="FiberPhotometryResponseSeries",
+        neurodata_type_inc="TimeSeries",
+        doc="Extends TimeSeries to hold Fiber Photometry data",
+        datasets=[
+            NWBDatasetSpec(
+                name="fiber",
+                doc="references row of FibersTable",
+                neurodata_type_inc="DynamicTableRegion",
+                quantity="?",
+            ),
+            NWBDatasetSpec(
+                name="excitation_source",
+                doc="references row of ExcitationSourcesTable",
+                neurodata_type_inc="DynamicTableRegion",
+                quantity="?",
+            ),
+            NWBDatasetSpec(
+                name="fluorophore",
```

Comment:
Similarly, one fiber photometry response series can arise from multiple different fluorophores. So this field should reference one or more rows of the fluorophores table.